Framework/Core/src/DataSamplingConditionRandom.cxx

```diff
@@ -40,10 +40,10 @@ class DataSamplingConditionRandom : public DataSamplingCondition
   ~DataSamplingConditionRandom() = default;
 
   /// \brief Reads 'fraction' parameter (type double, between 0 and 1) and seed (int).
-  void configure(const o2::configuration::tree::Branch& cfg) override
+  void configure(const boost::property_tree::ptree& cfg) override
   {
-    mFraction = configuration::tree::getRequired<double>(cfg, "fraction");
-    mSeed = configuration::tree::getRequired<int>(cfg, "seed");
+    mFraction = cfg.get<double>("fraction");
```

Comment:
Not sure this 'cfg' abbreviation follows the coding guidelines.

Reply:
I did not touch variable names. This can be addressed in separated PR.

Reply:
I found 'cfg' abbreviation quite obvious for any programmer, but I am completely fine with changing it to full 'configuration' (which doesn't seem to be ideal as well, since the namespace is named like that). Any other ideas?

Reply:
What about 'config' ?
It's also a sort of abbreviation, but one that we actually use verbally.
And maybe less ambiguous wrt 'cfg'.

Reply:
OK, I will be doing some improvements for random number generation soon, then I can address it on the occasion.